governance/assessments/projects/it-toto/2024-10-31.md

```diff
@@ -0,0 +1,118 @@
+
+**Election**
+The 2025 election is coming soon. The [election schedule](https://github.com/in-toto/community/blob/3da1c4ed10c78ab47523ddda9625fd9e3f4043be/STEERING-COMMITTEE.md) should be updated with the details. The project should also consider staggering terms to allow for continuity.
+
+**Linking Governance**
+archivista and witness have maintainers, but do not link back to the overall project governance.
+
+**Meeting Minutes**
+Meeting minutes have been moved out of the Github [repo](https://github.com/in-toto/community/tree/main/community-meetings) and to [hackmd](https://hackmd.io/Czv1Si4jQkieo_qByoMkPQ). This should be updated in the repo so people know how to join the community meeting.
+
+**Contributor Guides**
+There is an overall contributor [guide](https://github.com/in-toto/community/blob/main/CONTRIBUTING.md). Many of the implementations and sub-projects could benefit from more detailed contributing guides to get new contributors started, [witness](https://github.com/in-toto/witness/blob/main/CONTRIBUTING.md) is the best example of this.
+
+## Review
+
+### Governance Description
+
+it-toto switched to a steering committee style of governance roughly two years ago. The steering committee is made of 5 members from both academia and industry. Individual sub-project also have maintainers who manage merging code into the repo.
+
+### Discoverability
+
+#### Governance Location
+
+All governance documents are  in the community [repo](https://github.com/in-toto/community/tree/main) which is easy to find on Github. 
+
+#### Governance Discovery Completeness
+
+Governance documentation is found in many repos however is currently missing from archivista and witness.
+
+### Documentation Content
+
+| Governance Area | Coverage | Documents | Finding Notes |
+|:----------------|:--------:|:------:|:--------------|
+| Project Purpose | Complete | https://github.com/in-toto/community/blob/main/README.md | Purpose is very high level, it may be helpful to write a project vision to make it clear what is in and out of scope for the project |
+| Maintainer List | Complete | https://github.com/in-toto/community/blob/main/STEERING-COMMITTEE.md | The steering committee is not up to date with the CNCF project maintainers csv. Elections are also coming soon. |
+| Code of Conduct | Complete | https://github.com/in-toto/community/blob/main/CODE-OF-CONDUCT.md | |
+| Contributor Guide | Partial | https://github.com/in-toto/community/blob/main/CONTRIBUTING.md | The project has a general contributing guide, however many of the implementations do not. [witness](https://github.com/in-toto/witness/blob/main/CONTRIBUTING.md) is the best example with a guide on how to set up the development environment. |
+| Contributor Ladder | Partial | https://github.com/in-toto/community/blob/main/GOVERNANCE.md#project-roles | There are currently only contributors and maintainers. It may be helpful to lay out attitional steps between them so people can "climb the ladder". |
```

Comment:
```suggestion
| Contributor Ladder | Partial | https://github.com/in-toto/community/blob/main/GOVERNANCE.md#project-roles | There are currently only contributors and maintainers. It may be helpful to lay out additional steps between them so people can "climb the ladder". |
```